010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,119 @@
-
-As part of our commitment to securing data access and storage, Xata encrypts user data content:
-
-- during network transit with Transport Layer Security
-- at rest with storage encryption provided by our underlying infrastructure providers
-
-The same level of security standard applies to internal backups, logs and traces which do not expose plain text data content to third parties.
-
-For disaster recovery and change control purposes, Xata performs frequent backups of user data which is retained only as required and while it continues to have a legitimate purpose for delivering our services to the user.
-
-## Conclusions and design principles
-
-At Xata we are taking hard data problems, solve them using the industry's best practices, and then offer them to you in the most easy-to-use way possible.
-
-Zero-downtime migrations are an example of this: difficult under the hood, but a familiar-looking pull request with a diff in our UI. Change Data Capture is another example: generally difficult to get right, but inside Xata your data is simply replicated to the search engine, with zero configuration.
-
-Our design principles are:
-
-- **Use proven technology**, best-in-class for a particular data problem. This is the reason we use PostgreSQL for the main data store and Elasticsearch for search and analytics. For edge functions, we rely on Cloudflare Workers, which has a wide and reliable network. Because we use proven technology, the resulting service is reliable and your data is safe.
-- **Build for vertical integration** for a smooth developer experience. Besides the backend service, we put a lot of effort into our admin UI and SDKs in multiple languages. They work together without any friction.
-- **Cloud-only and serverless**. Operating and monitoring databases is hard, especially when dealing with multiple of them. We are taking as much load off your shoulders as we can. This includes high availability, maintaining replicas, global distributions, etc. This blog post goes into detail on what we mean when we talk about [serverless databases](https://xata.io/blog/what-is-a-serverless-database).
-- **Make the complex simple**. We’re leveraging things like the UI, rich data types, and high-level SDK in order to to make Xata the easiest database to use. At the same time, we’re not sacrificing power, and our functionality goes deep in aspects like search and analytics.
-
-Xata is a relatively new service and we have a lot of ideas for improving it. For example, we plan to add more data stores and more ways to take advantage of the existing data stores. But you can expect us to follow the principles above.
+The Xata SDKs support all three methods above, so it easy to switch between them depending on the use case. The SDKs are available for multiple programming languages, including TypeScript, JavaScript, Python, and Go and are continuously tested with the Xata service.
```

Comment:
Would it be weird to end with a "get started today" CTA to sign up from docs?

Reply:
We always have the "Start free" CTA in the bar.